fiduceo_uncertainties.f90

```diff
@@ -1416,12 +1447,6 @@ SUBROUTINE radiance_uncertainties(i,outData,coefs1,coefs2,coefs3,FCDR,&
             FCDR%uce3(outData%nelem,outData%arraySize),&
             FCDR%uce4(outData%nelem,outData%arraySize),&
             FCDR%uce5(outData%nelem,outData%arraySize),&
-            FCDR%ucict3(outdata%arraySize),&
```

Comment:
Needed for correct handling of uncertainties

Reply:
Correct. I have added back in the following:

-->
            FCDR%ucict3(outdata%arraySize),&
            FCDR%ucict4(outdata%arraySize),&
            FCDR%ucict5(outdata%arraySize),&
            FCDR%ucs3(outdata%arraySize),&
            FCDR%ucs4(outdata%arraySize),&
            FCDR%ucs5(outdata%arraySize),&